test.sh

```diff
@@ -12,6 +12,10 @@ run_with_suppressed_output () {
 	fi
 }
 
+# Use printf to get appropriate version string for comparison
+get_version_string() {
+        printf "%02d%02d%02d" $(echo "$1" | tr '.' ' ');
```

Comment:
```suggestion
	printf "%02d%02d%02d" $(echo "$1" | tr '.' ' ');
```

Rather than copying this into two files, can you please put it in https://github.com/disney/meteor-base/blob/main/support.sh? That’s the file with all the shared code.

Reply:
Thanks, I've moved the function to support.sh.